### PR TITLE
Update install with step #s and kernel download

### PR DESCRIPTION
**Pull request by mattwojo**

### CODE REVIEW

**Comment by mattwojo** [comment]
Hold off on merging -- still needs a review and a few updates.

**Comment by mattwojo** [comment]
- Add step #s to H2s on WSL install page
- Fix any updated H2 anchor links
- Move kernel package download instructions into main WSL install page
- Delete kernel install page
- Move 'What's new in WSL 2' and 'About WSL 2' content into 'Compare WSL 1 and WSL 2' page
- Delete About WSL 2 page
- Delete Whats new in WSL 2 page
- Set up redirects for deleted pages
- Delete Install WSL - archive file

File: WSL/install-win10.md [hunk 1]
```diff
@@ -1,40 +1,48 @@
...
-To update to WSL 2, you must meet the following criteria:
+### Requirements
 
-- Running Windows 10, [updated to version 1903 or higher](ms-settings:windowsupdate), **Build 18362** or higher for x64 systems.
-   - If you are on Windows 10 version 1903 or 1909 make sure your minor build number is 1049 or higher. See [full troubleshooting instructions here](https://docs.microsoft.com/windows/wsl/troubleshooting#im-on-windows-10-version-1903-and-i-still-do-not-see-options-for-wsl-2)
-- Running Windows 10, updated to version 2004 or higher, **build 19041**, for ARM64 systems.
-- Please note if you are on Windows 10 version 1903 or 1909 you will need to ensure that you have the proper backport, instructions can be [found here](https://devblogs.microsoft.com/commandline/wsl-2-support-is-coming-to-windows-10-versions-1903-and-1909/#how-do-i-get-it). 
+- For x64 systems: **Version 1903** or higher, with **Build 18362** or higher.
+- For ARM64 systems: **Version 2004** or higher, with **Build 19041** or higher.
+- Builds lower than 18362 do not support WSL 2. Use the [Windows Update Assistant](https://www.microsoft.com/software-download/windows10).
```
Comment (craigloewen-msft) [inline WSL/install-win10.md:36]: @mattwojo It says 'Use the UPdate Assistant', but to do what? To update?

Comment (mattwojo) [reply WSL/install-win10.md:36]: I'll update to: "Use the Windows Update Assistant to update your version of Windows." Sound ok?

**Pull request merged**